### DISCUSSION TITLE
var 3.0 Err : Value cannot be null. (Parameter 'PART_MainGrid is null'

### DISCUSSION THREAD

**Discussion by noet-Leesj** [lepoco/wpfui#1032] [Q&A]
![image](https://github.com/lepoco/wpfui/assets/90229133/7e41533b-5569-4584-a245-e8a0820df3c5)

I'm trying to use version 3.0 and I got an error. I think it's a Title bar Error, is there an error fix?

If I don't use the title bar, it runs without errors.

**Comment by cgsrogers** [comment]
Did you ever figure this out? I think I have a similar issue
